### DISCUSSION TITLE
Question: Is there a tutorial or plug-in made on top of pixi.js to help me create a graphic user interface?

### DISCUSSION THREAD

**Discussion by inglesuniversal** [pixijs/pixijs#7188] [Q&A]
Hello, I am new using Pixi.js and I wish to know if there is an interface built around it where I can create a graphic interface with buttons, sliders, virtual keyboard and position those items on a playground-like interface and the get the coordinates or code to use it on my project?

I found a couple of repositories but they seem to be abandoned projects.

Thanks and #StaySafe

**Comment by xzilja** [comment]
I think best way is to overlay dom elements over pixi canvas for this, however there are also "experimental" libraries like https://github.com/pixijs/pixi-ui that provide access to some common ui / layout options that are rendered by pixi.
